src/intelfocl/intelfocl_device.cc

```diff
@@ -0,0 +1,185 @@
+#include <numeric>
+#include <dmlc/logging.h>
+#include <vta/hw_spec.h>
+#include "intelfocl_device.h"
+#include "aoclutils/aocl_utils.h"
+
+#define CL_STATUS_SUCCESS(x) ((x) == CL_SUCCESS)
+
+void cleanup() {}
```

Comment:
actually clean up?

Reply:
This cleanup() function is a callback function from 3rdparty code aoclutils, it gives the user a chance to perform cleanups before calling exit(), if required.
Here our code does not require any special cleanup before exit(), thus the cleanup() was left blank by intention.

Reply:
@remotego could you add a comment in the code explaining what you just mentioned? thank you